### PR TITLE
Fix PrimaryButton with url color

### PR DESCRIPTION
**Pull request by GeorgiSTodorov**
Make sure to test in both Firefox and Chrome as some of the solutions that I tried worked only in Chrome.

### Before
Screenshot from 2020-03-09 14-53-50

### After
Screenshot from 2020-03-09 14-44-31

### CODE REVIEW

File: packages/button/stories/DefaultButton.stories.tsx [hunk 1]
```diff
@@ -242,6 +242,46 @@ storiesOf("Buttons/Default", module)
       }
     }
   )
+  .add(
+    "PrimaryButton used as a link",
```
Comment (mperrotti) [inline packages/button/stories/DefaultButton.stories.tsx]: I don't think this story is necessary. If we want to show multiple button appearances being used as a link, I think we should do it in one story

File: packages/button/components/ButtonBase.tsx [hunk 2]
```diff
@@ -112,7 +115,7 @@ class ButtonBase extends React.PureComponent<ButtonBaseProps, {}> {
       textWeight("medium"),
       className,
       {
-        [linkReset]: url,
+        [appearance === "primary" ? primaryLinkReset : linkReset]: url,
```
Comment (mperrotti) [inline packages/button/components/ButtonBase.tsx]: I figured out a simpler way to do this. I'll push up a commit.

File: packages/button/stories/DefaultButton.stories.tsx (hunk 1 above)
Comment (GeorgiSTodorov) [reply packages/button/stories/DefaultButton.stories.tsx]: The problem is that PrimaryButton used as a link didn't have the expected color.

**Comment by mperrotti** [comment]
Ignore my last commit, I screwed something up. I'll push a fix

File: packages/button/stories/DefaultButton.stories.tsx (hunk 1 above)
Comment (mperrotti) [reply packages/button/stories/DefaultButton.stories.tsx]: I understand why you're showing the PrimaryButton with a `url` prop, but I think we should just use one story to show different button styles with `url`.

Comment (GeorgiSTodorov) [reply packages/button/stories/DefaultButton.stories.tsx]: Okay, I agree.

Comment (mperrotti) [reply packages/button/stories/DefaultButton.stories.tsx]: I pushed a commit to take care of Storybook updates.

**Pull request merged**